velox/dwio/parquet/tests/ParquetReaderTest.cpp

```diff
@@ -426,6 +426,30 @@ TEST_F(ParquetReaderTest, varcharFilters) {
   assertReadWithFilters(
       "nation.parquet", rowType, std::move(filters), expected);
 
-  // TODO Test name IN ('CANADA', 'UNITED KINGDOM') filter. Currently it doesn't
-  // work. The reader returns all rows instead of just 2.
+  // name IN ('CANADA', 'UNITED KINGDOM')
+  const std::vector<std::string> twoFilterValues{"CANADA", "UNITED KINGDOM"};
```

Comment:
nit: twoFilterValues name is not very descriptive. Since it is used only once, I'd inline it, e.g. 

```
filters.insert({"name", common::test::in({"CANADA", "UNITED KINGDOM")});
```

Reply:
I could not inline when I overloaded `equal` before. Works with `in`. Thanks.